mlir/include/mlir/Dialect/Linalg/TransformOps/LinalgTransformOps.td

```diff
@@ -23,7 +23,7 @@ include "mlir/IR/RegionKindInterface.td"
 // value in the payload IR.
 def TransformParamTypeOrAnyHandle : Type<
     Or<[TransformHandleTypeInterface.predicate,
-        Transform_ParamType.predicate]>,
+        TransformParamTypeInterface.predicate]>,
```

Comment:
Nit: this type constraint is called `TransformParamTypeOrAnyHandle`, indicating that it accepts either `!transform.param` type specifically or any type implementing `TransformHandleTypeInterface`. This change will make it also accept any type implementing `TransformParamTypeInterface`, so it no longer matches the name. Consider renaming to `TransformAnyParamOrOpHandle`.